### PR TITLE
Add Acheron Themis and its variants

### PR DESCRIPTION
**Pull request by Gondolindrim**
## Description

Adds support for Acheron Themis and its variants.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/acheron/themis/readme.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Acheron Themis QMK firmware
+
+![athena](https://acheronproject.com/images/athena/IMG_9653.jpg)
```
Comment (dunk2k) [inline keyboards/acheron/themis/readme.md:3]: image doesn't render 🙁 - please re-host on Imgur.

File: keyboards/acheron/themis/87h/info.json [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+        "audio":     false,
+        "encoder":   false,
+        "nkro":      true
+    },
+    "diode_direction": "COL2ROW",
+    "ws2812": {
+        "pin": "B15"
+    },
+    "processor": "STM32F401",
+    "bootloader": "stm32-dfu",
```
Comment (dunk2k) [inline keyboards/acheron/themis/87h/info.json:28]:
```suggestion
    "bootloader": "stm32-dfu",
    "community_layouts": ["tkl_ansi"],
```

File: keyboards/acheron/themis/87htsc/info.json [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+        "nkro":      true
+    },
+    "diode_direction": "COL2ROW",
+    "ws2812": {
+        "pin": "B15"
+    },
+    "processor": "STM32F401",
+    "bootloader": "stm32-dfu",
+    "layouts": {
+        "LAYOUT_tkl_ansi": {
```
Comment (dunk2k) [inline keyboards/acheron/themis/87htsc/info.json:30]:
```suggestion
    "community_layouts": ["tkl_ansi_tsangan"],
    "layouts": {
        "LAYOUT_tkl_ansi_tsangan": {
```

File: keyboards/acheron/themis/87htsc/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+   [0] = LAYOUT_tkl_ansi(
+        KC_ESC , KC_F1  , KC_F2  , KC_F3  , KC_F4  , KC_F5  , KC_F6  , KC_F7  , KC_F8  , KC_F9  , KC_F10 , KC_F11 , KC_F12 ,          KC_PSCR, KC_SCRL, KC_PAUS , 
+        KC_GRV , KC_1   , KC_2   , KC_3   , KC_4   , KC_5   , KC_6   , KC_7   , KC_8   , KC_9   , KC_0   , KC_MINS, KC_EQL , KC_BSPC, KC_INS , KC_HOME, KC_PGUP ,
+        KC_TAB , KC_Q   , KC_W   , KC_E   , KC_R   , KC_T   , KC_Y   , KC_U   , KC_I   , KC_O   , KC_P   , KC_LBRC, KC_RBRC, KC_BSLS, KC_DEL , KC_END , KC_PGDN ,
+        KC_CAPS, KC_A   , KC_S   , KC_D   , KC_F   , KC_G   , KC_H   , KC_J   , KC_K   , KC_L   , KC_SCLN, KC_QUOT, KC_ENT ,
+	KC_LSFT,          KC_Z   , KC_X   , KC_C   , KC_V   , KC_B   , KC_N   , KC_M   , KC_COMM, KC_DOT , KC_SLSH, KC_RSFT,                   KC_UP  ,
+        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC ,                            KC_RALT, KC_RGUI, KC_RCTL,          KC_LEFT, KC_DOWN, KC_RGHT
+	),
+   [1] = LAYOUT_tkl_ansi(
```
Comment (dunk2k) [inline keyboards/acheron/themis/87htsc/keymaps/default/keymap.c:29]:
```suggestion
   [0] = LAYOUT_tkl_ansi_tsangan(
        KC_ESC , KC_F1  , KC_F2  , KC_F3  , KC_F4  , KC_F5  , KC_F6  , KC_F7  , KC_F8  , KC_F9  , KC_F10 , KC_F11 , KC_F12 ,          KC_PSCR, KC_SCRL, KC_PAUS , 
        KC_GRV , KC_1   , KC_2   , KC_3   , KC_4   , KC_5   , KC_6   , KC_7   , KC_8   , KC_9   , KC_0   , KC_MINS, KC_EQL , KC_BSPC, KC_INS , KC_HOME, KC_PGUP ,
        KC_TAB , KC_Q   , KC_W   , KC_E   , KC_R   , KC_T   , KC_Y   , KC_U   , KC_I   , KC_O   , KC_P   , KC_LBRC, KC_RBRC, KC_BSLS, KC_DEL , KC_END , KC_PGDN ,
        KC_CAPS, KC_A   , KC_S   , KC_D   , KC_F   , KC_G   , KC_H   , KC_J   , KC_K   , KC_L   , KC_SCLN, KC_QUOT, KC_ENT ,
	KC_LSFT,          KC_Z   , KC_X   , KC_C   , KC_V   , KC_B   , KC_N   , KC_M   , KC_COMM, KC_DOT , KC_SLSH, KC_RSFT,                   KC_UP  ,
        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC ,                            KC_RALT, KC_RGUI, KC_RCTL,          KC_LEFT, KC_DOWN, KC_RGHT
	),
   [1] = LAYOUT_tkl_ansi_tsangan(
```

File: keyboards/acheron/themis/87htsc/keymaps/via/keymap.c [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+   [0] = LAYOUT_tkl_ansi(
+        KC_ESC , KC_F1  , KC_F2  , KC_F3  , KC_F4  , KC_F5  , KC_F6  , KC_F7  , KC_F8  , KC_F9  , KC_F10 , KC_F11 , KC_F12 ,          KC_PSCR, KC_SCRL, KC_PAUS , 
+        KC_GRV , KC_1   , KC_2   , KC_3   , KC_4   , KC_5   , KC_6   , KC_7   , KC_8   , KC_9   , KC_0   , KC_MINS, KC_EQL , KC_BSPC, KC_INS , KC_HOME, KC_PGUP ,
+        KC_TAB , KC_Q   , KC_W   , KC_E   , KC_R   , KC_T   , KC_Y   , KC_U   , KC_I   , KC_O   , KC_P   , KC_LBRC, KC_RBRC, KC_BSLS, KC_DEL , KC_END , KC_PGDN ,
+        KC_CAPS, KC_A   , KC_S   , KC_D   , KC_F   , KC_G   , KC_H   , KC_J   , KC_K   , KC_L   , KC_SCLN, KC_QUOT, KC_ENT ,
+	KC_LSFT,          KC_Z   , KC_X   , KC_C   , KC_V   , KC_B   , KC_N   , KC_M   , KC_COMM, KC_DOT , KC_SLSH, KC_RSFT,                   KC_UP  ,
+        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC ,                            KC_RALT, KC_RGUI, KC_RCTL,          KC_LEFT, KC_DOWN, KC_RGHT
+	),
+   [1] = LAYOUT_tkl_ansi(
```
Comment (dunk2k) [inline keyboards/acheron/themis/87htsc/keymaps/via/keymap.c:29]:
```suggestion
   [0] = LAYOUT_tkl_ansi_tsangan(
        KC_ESC , KC_F1  , KC_F2  , KC_F3  , KC_F4  , KC_F5  , KC_F6  , KC_F7  , KC_F8  , KC_F9  , KC_F10 , KC_F11 , KC_F12 ,          KC_PSCR, KC_SCRL, KC_PAUS , 
        KC_GRV , KC_1   , KC_2   , KC_3   , KC_4   , KC_5   , KC_6   , KC_7   , KC_8   , KC_9   , KC_0   , KC_MINS, KC_EQL , KC_BSPC, KC_INS , KC_HOME, KC_PGUP ,
        KC_TAB , KC_Q   , KC_W   , KC_E   , KC_R   , KC_T   , KC_Y   , KC_U   , KC_I   , KC_O   , KC_P   , KC_LBRC, KC_RBRC, KC_BSLS, KC_DEL , KC_END , KC_PGDN ,
        KC_CAPS, KC_A   , KC_S   , KC_D   , KC_F   , KC_G   , KC_H   , KC_J   , KC_K   , KC_L   , KC_SCLN, KC_QUOT, KC_ENT ,
	KC_LSFT,          KC_Z   , KC_X   , KC_C   , KC_V   , KC_B   , KC_N   , KC_M   , KC_COMM, KC_DOT , KC_SLSH, KC_RSFT,                   KC_UP  ,
        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC ,                            KC_RALT, KC_RGUI, KC_RCTL,          KC_LEFT, KC_DOWN, KC_RGHT
	),
   [1] = LAYOUT_tkl_ansi_tsangan(
```

File: keyboards/acheron/themis/88htsc/info.json [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+        "nkro":      true
+    },
+    "diode_direction": "COL2ROW",
+    "ws2812": {
+        "pin": "B15"
+    },
+    "processor": "STM32F401",
+    "bootloader": "stm32-dfu",
+    "layouts": {
+        "LAYOUT_tkl_ansi": {
```
Comment (dunk2k) [inline keyboards/acheron/themis/88htsc/info.json:30]:
```suggestion
    "community_layouts": ["tkl_f13_ansi_tsangan"],
    "layouts": {
        "LAYOUT_tkl_f13_ansi_tsangan": {
```

File: keyboards/acheron/themis/88htsc/keymaps/default/keymap.c [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+   [0] = LAYOUT_tkl_ansi(
+        KC_ESC , KC_F1  , KC_F2  , KC_F3  , KC_F4  , KC_F5  , KC_F6  , KC_F7  , KC_F8  , KC_F9  , KC_F10 , KC_F11 , KC_F12 , KC_F13,  KC_PSCR, KC_SCRL, KC_PAUS , 
+        KC_GRV , KC_1   , KC_2   , KC_3   , KC_4   , KC_5   , KC_6   , KC_7   , KC_8   , KC_9   , KC_0   , KC_MINS, KC_EQL , KC_BSPC, KC_INS , KC_HOME, KC_PGUP ,
+        KC_TAB , KC_Q   , KC_W   , KC_E   , KC_R   , KC_T   , KC_Y   , KC_U   , KC_I   , KC_O   , KC_P   , KC_LBRC, KC_RBRC, KC_BSLS, KC_DEL , KC_END , KC_PGDN ,
+        KC_CAPS, KC_A   , KC_S   , KC_D   , KC_F   , KC_G   , KC_H   , KC_J   , KC_K   , KC_L   , KC_SCLN, KC_QUOT, KC_ENT ,
+	KC_LSFT,          KC_Z   , KC_X   , KC_C   , KC_V   , KC_B   , KC_N   , KC_M   , KC_COMM, KC_DOT , KC_SLSH, KC_RSFT,                   KC_UP  ,
+        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC ,                            KC_RALT, KC_RGUI, KC_RCTL,          KC_LEFT, KC_DOWN, KC_RGHT
+	),
+   [1] = LAYOUT_tkl_ansi(
```
Comment (dunk2k) [inline keyboards/acheron/themis/88htsc/keymaps/default/keymap.c:29]:
```suggestion
   [0] = LAYOUT_tkl_f13_ansi_tsangan(
        KC_ESC , KC_F1  , KC_F2  , KC_F3  , KC_F4  , KC_F5  , KC_F6  , KC_F7  , KC_F8  , KC_F9  , KC_F10 , KC_F11 , KC_F12 , KC_F13,  KC_PSCR, KC_SCRL, KC_PAUS , 
        KC_GRV , KC_1   , KC_2   , KC_3   , KC_4   , KC_5   , KC_6   , KC_7   , KC_8   , KC_9   , KC_0   , KC_MINS, KC_EQL , KC_BSPC, KC_INS , KC_HOME, KC_PGUP ,
        KC_TAB , KC_Q   , KC_W   , KC_E   , KC_R   , KC_T   , KC_Y   , KC_U   , KC_I   , KC_O   , KC_P   , KC_LBRC, KC_RBRC, KC_BSLS, KC_DEL , KC_END , KC_PGDN ,
        KC_CAPS, KC_A   , KC_S   , KC_D   , KC_F   , KC_G   , KC_H   , KC_J   , KC_K   , KC_L   , KC_SCLN, KC_QUOT, KC_ENT ,
	KC_LSFT,          KC_Z   , KC_X   , KC_C   , KC_V   , KC_B   , KC_N   , KC_M   , KC_COMM, KC_DOT , KC_SLSH, KC_RSFT,                   KC_UP  ,
        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC ,                            KC_RALT, KC_RGUI, KC_RCTL,          KC_LEFT, KC_DOWN, KC_RGHT
	),
   [1] = LAYOUT_tkl_f13_ansi_tsangan(
```

File: keyboards/acheron/themis/88htsc/keymaps/via/keymap.c [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+   [0] = LAYOUT_tkl_ansi(
+        KC_ESC , KC_F1  , KC_F2  , KC_F3  , KC_F4  , KC_F5  , KC_F6  , KC_F7  , KC_F8  , KC_F9  , KC_F10 , KC_F11 , KC_F12 , KC_F13,  KC_PSCR, KC_SCRL, KC_PAUS , 
+        KC_GRV , KC_1   , KC_2   , KC_3   , KC_4   , KC_5   , KC_6   , KC_7   , KC_8   , KC_9   , KC_0   , KC_MINS, KC_EQL , KC_BSPC, KC_INS , KC_HOME, KC_PGUP ,
+        KC_TAB , KC_Q   , KC_W   , KC_E   , KC_R   , KC_T   , KC_Y   , KC_U   , KC_I   , KC_O   , KC_P   , KC_LBRC, KC_RBRC, KC_BSLS, KC_DEL , KC_END , KC_PGDN ,
+        KC_CAPS, KC_A   , KC_S   , KC_D   , KC_F   , KC_G   , KC_H   , KC_J   , KC_K   , KC_L   , KC_SCLN, KC_QUOT, KC_ENT ,
+	KC_LSFT,          KC_Z   , KC_X   , KC_C   , KC_V   , KC_B   , KC_N   , KC_M   , KC_COMM, KC_DOT , KC_SLSH, KC_RSFT,                   KC_UP  ,
+        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC ,                            KC_RALT, KC_RGUI, KC_RCTL,          KC_LEFT, KC_DOWN, KC_RGHT
+	),
+   [1] = LAYOUT_tkl_ansi(
```
Comment (dunk2k) [inline keyboards/acheron/themis/88htsc/keymaps/via/keymap.c:29]:
```suggestion
   [0] = LAYOUT_tkl_f13_ansi_tsangan(
        KC_ESC , KC_F1  , KC_F2  , KC_F3  , KC_F4  , KC_F5  , KC_F6  , KC_F7  , KC_F8  , KC_F9  , KC_F10 , KC_F11 , KC_F12 , KC_F13,  KC_PSCR, KC_SCRL, KC_PAUS , 
        KC_GRV , KC_1   , KC_2   , KC_3   , KC_4   , KC_5   , KC_6   , KC_7   , KC_8   , KC_9   , KC_0   , KC_MINS, KC_EQL , KC_BSPC, KC_INS , KC_HOME, KC_PGUP ,
        KC_TAB , KC_Q   , KC_W   , KC_E   , KC_R   , KC_T   , KC_Y   , KC_U   , KC_I   , KC_O   , KC_P   , KC_LBRC, KC_RBRC, KC_BSLS, KC_DEL , KC_END , KC_PGDN ,
        KC_CAPS, KC_A   , KC_S   , KC_D   , KC_F   , KC_G   , KC_H   , KC_J   , KC_K   , KC_L   , KC_SCLN, KC_QUOT, KC_ENT ,
	KC_LSFT,          KC_Z   , KC_X   , KC_C   , KC_V   , KC_B   , KC_N   , KC_M   , KC_COMM, KC_DOT , KC_SLSH, KC_RSFT,                   KC_UP  ,
        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC ,                            KC_RALT, KC_RGUI, KC_RCTL,          KC_LEFT, KC_DOWN, KC_RGHT
	),
   [1] = LAYOUT_tkl_f13_ansi_tsangan(
```

**Review by dunk2k — CHANGES_REQUESTED** [review]

File: keyboards/acheron/themis/readme.md [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+
+The Themis is Acheron Project's open-source, freely available tenkeyless (TKL) keyboard Printed Circuit Boards (PCB) which features:
+
+- No LED lighting
+- Solderless ("hotswap") MX switches support
+- Fixed layout for each variant
+
+* Keyboard Maintainer: [Gondolindrim](https://github.com/Gondolindrim)
+* Hardware Supported: open-source PCB using the [Joker template](https://acheronproject.com/joker_mcus/joker/). First units were manufactured using STM32F4x1 microcontroller;
+* Hardware Availability: these PCBs are open-source so you can manufacture them yourself or order them from a factory. See the [project's documentation](https://acheronproject.com/pcbs/athena/athena/) for more information.
```
Comment (wlhlm) [inline keyboards/acheron/themis/readme.md:13]: Link returns 404.

File: keyboards/acheron/themis/87h/rules.mk [hunk 10]
```diff
@@ -0,0 +1,2 @@
+# Enter lower-power sleep mode when on the ChibiOS idle thread
+OPT_DEFS += -DCORTEX_ENABLE_WFI_IDLE=TRUE
```
Comment (drashna) [inline keyboards/acheron/themis/87h/rules.mk:2]:
```suggestion
```

File: keyboards/acheron/themis/87htsc/rules.mk [hunk 11]
```diff
@@ -0,0 +1,2 @@
+# Enter lower-power sleep mode when on the ChibiOS idle thread
+OPT_DEFS += -DCORTEX_ENABLE_WFI_IDLE=TRUE
```
Comment (drashna) [inline keyboards/acheron/themis/87htsc/rules.mk:2]:
```suggestion
```

File: keyboards/acheron/athena/alpha/alpha.h
Comment (drashna) [inline keyboards/acheron/athena/alpha/alpha.h:1]: This is already in info.json.  This file should be removed.

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/acheron/themis/readme.md (hunk 9 above)
Comment (Gondolindrim) [reply keyboards/acheron/themis/readme.md]: Fixed in 2ff0be2

File: keyboards/acheron/themis/readme.md (hunk 1 above)
Comment (Gondolindrim) [reply keyboards/acheron/themis/readme.md]: Fixed in 2ff0be2

**Comment by Gondolindrim** [comment]
Reopened a new branch and reapplied changes to reset the folder `acheron/athena`. It was not supposed to be in this PR, sorry for this.

File: keyboards/acheron/themis/readme.md [hunk 12]
```diff
@@ -0,0 +1,23 @@
+# Acheron Themis QMK firmware
+
+![themis](https://imgur.com/kwemOsJ.png)
```
Comment (drashna) [inline keyboards/acheron/themis/readme.md:3]:
```suggestion
![themis](https://i.imgur.com/kwemOsJh.png)
```

**Review by drashna — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**